### DISCUSSION TITLE
Folding Range

### DISCUSSION THREAD

**Discussion by Yudha17** [microsoft/vscode-discussions#622] [Extension Development QnA] [answered]
How if I meet the folding range for 65000 as a maximum folding regions ? What is the solution ? Can it set to be higher ?

**Comment by apupier** [comment, marked as answer]
In 1.73, the maximum was set to 5000 https://code.visualstudio.com/updates/v1_73#_no-more-too-many-folding-ranges-notifications . it can be modified in User settings

![image](https://github.com/microsoft/vscode-discussions/assets/1105127/00f78ef6-3dbd-42c6-b2eb-527f6cd2f62f)
